jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -384,8 +384,7 @@ def __init__(self, *args, **kwargs):
 
     @property
     def user_api(self):
-        return PluginUserApi(self, expose=('viewer',
-                                           'interactive_extract',
+        return PluginUserApi(self, expose=('interactive_extract',
```

Comment:
I'm actually not sure why this wasn't tripping a test failure before now... even on main, `viewer` does not exist in spectral extraction as far as I can tell 🤷‍♂️